snow/networking/handler/handler.go

```diff
@@ -844,18 +853,30 @@ func (h *handler) handleChanMsg(msg message.InboundMessage) error {
 		zap.Any("message", body),
 	)
 	h.ctx.Lock.Lock()
+	lockAcquiredTime := h.clock.Time()
```

Comment:
Maybe we should call this `lockHeldStartTime`; `lockAcquireTime` sounds more like the time waiting to grab a lock

Reply:
I think because it is `Acquired` (past tense) it's pretty clear.